src/components/Buttons/SplitButton.vue

```diff
@@ -31,31 +31,44 @@ const handleSubClick = () => {
   rotateDropDown.value = false;
   emit('subClick');
 };
+
+const buttonHeightClass = computed(() => {
+  switch (props.size) {
+    case 'small':
+      return 'h-8';
+    case 'medium':
+      return 'h-10';
+    case 'large':
+      return 'h-12';
+    default:
+      return 'h-10';
+  }
+});
 </script>
 
 <template>
   <n-button-group>
     <Button
       v-if="mainText"
       :href="props.href"
-      :to="props.to"
       :color="props.mainColor"
+      :size="$props.size"
       @click="handleMainClick"
     >
       {{ props.mainText }}
     </Button>
 
     <n-popover ref="popoverRef" trigger="click" raw placement="bottom" @update:show="onUpdateShow">
       <template #trigger>
-        <Button class="h-10 flex items-center">
+        <Button :size="$props.size" :class="buttonHeightClass">
```

Comment:
No need for `flex items-center`any longer?

Reply:
No difference at all.